lib/core-integration/src/Test/Integration/Scenario/CLI/Shelley/HWWallets.hs

```diff
@@ -122,13 +124,16 @@ spec = describe "SHELLEY_CLI_HW_WALLETS" $ do
         _ <- expectValidJSON (Proxy @(ApiTransaction n)) op
         cp `shouldBe` ExitSuccess
 
-        eventually "Wallet balance is as expected" $ do
```

Comment:
I was thinking that it would be a shame to lose these high-level functional descriptions of that conditions are being tested.

So I have added a `verifyMsg` function which includes such a description.